samples/ImageSharp.Web.Sample/Pages/_ViewImports.cshtml

```diff
@@ -0,0 +1,7 @@
+@using ImageSharp.Web.Sample
```

Comment:
It would be nice if we could add these namespaces at startup however, I don't think it's possible.

Reply:
You can add [implicit/global usings](https://learn.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates#implicit-using-directives) by providing a build target file that contains:
```xml
<ItemGroup>
  <Using Include="SixLabors.ImageSharp" />
  <Using Include="SixLabors.ImageSharp.Processing" />
  <Using Include="SixLabors.ImageSharp.Web" />
</ItemGroup>
```

Umbraco does this as well: https://github.com/umbraco/Umbraco-CMS/blob/3873a0735cde27bfd9692f4c7370613730ef4fd0/src/Umbraco.Cms.Targets/buildTransitive/Umbraco.Cms.Targets.props#L8-L11.

Reply:
I'm a daft one. I already know this! 🤣

https://github.com/SixLabors/ImageSharp/blob/451a713fc11f2825946350c8f02c25b310509f57/SixLabors.ImageSharp.props

Reply:
Done. Those includes in Umbraco should only be enabled if `ImplicitUsings` is enabled btw.

Reply:
Sometimes you just need a quick reminder! 😉 And thanks for the suggestion, I've just created a PR to fix this in Umbraco: https://github.com/umbraco/Umbraco-CMS/pull/13322 💪🏻